### ISSUE TITLE
[Bug]: flatpickrWrapper is not defined

### ISSUE THREAD

**Issue by sturlath** [Megabit/Blazorise#5370]
### Blazorise Version

1.4.2

### What Blazorise provider are you running on?

Bootstrap5

### Link to minimal reproduction, or a simple code snippet

I´m getting this error in the browser console my abp.io version 8.0.2. Everything looks to be working but I wanted to report this anyways..

```
datePicker.js?v=1.4.2.0:118  Uncaught ReferenceError: flatpickrWrapper is not defined
    at ClassWatcher.successClassAddHandler [as classAddedCallback] (datePicker.js?v=1.4.2.0:118:17)
    at MutationObserver.mutationCallback (observer.js?v=1.4.2.0:90:30)
```

This is the code I have around the DatePicker. I can´t see what is wrong if any

```charp
<Column>
    <Field>
        <FieldLabel>@L["StartingDate"]</FieldLabel>
        <Validation Validator="@ValidateEventStartingDate">
            <Addons>
                <Addon AddonType="AddonType.Body">
                    <DatePicker @ref="@startingDatePicker" TValue="DateTime?" InputMode="DateInputMode.Date" Date="@NewStreamEvent.StartingDate" DateChanged="@OnStartingDateChanged" DisplayFormat="dd-MM-Y" Disabled="@DisabledMode">
                    </DatePicker>
                </Addon>
                <Addon AddonType="AddonType.End">
                    <Button Color="Color.Light" Clicked="@(()=>startingDatePicker.ToggleAsync())">
                        <Icon Name="IconName.CalendarDay" />
                    </Button>
                </Addon>
                <ValidationError>@L["PleaseSelectValidFutureStartingDateForEventDate"]</ValidationError>
            </Addons>
        </Validation>
    </Field>
</Column>
```

### Steps to reproduce

See code above

### What is expected?

No error in console

### What is actually happening?

Error in console

### What browsers are you seeing the problem on?

Microsoft Edge

**Comment by stsrki** [comment]
This should be fixed with #5363

It will be available in 1.5 release, March 18th,

**Comment by sturlath** [comment]
Ahh.. I noticed this last week and looked for an issue then but didnt have time to report it then and didnt search now 🙈 but great its fixed
